### PR TITLE
Add pipeline-only changes for release notes and labels for changelogs

### PR DESCRIPTION
**Pull request by emilymye**
Relies on https://github.com/GoogleCloudPlatform/magic-modules/pull/1869

### CODE REVIEW

File: .ci/magic-modules/pyutils/downstreams_test.py [hunk 1]
```diff
@@ -62,8 +62,8 @@ def test_downstream_urls(self):
     test_client = Github(os.environ.get(TOKEN_ENV_VAR))
     result = get_downstream_urls(self.test_client,self.TEST_PR_NUM)
 
-    expected_cnt = len(self.EXPECTED_DOWNSTREAM_URLS)
-    self.assertEquals(len(result), expected_cnt,
+    expected_len = len(self.EXPECTED_DOWNSTREAM_URLS)
```
Comment (emilymye) [inline .ci/magic-modules/pyutils/downstreams_test.py]: @chrisst 👀

File: .ci/magic-modules/pyutils/strutils.py [hunk 2]
```diff
@@ -35,4 +37,73 @@ def parse_github_url(gh_url):
...
+  release_note = release_note.strip()
+  if release_note:
+    edited += "\n```releasenote\n%s\n```\n" % release_note.strip()
+  return edited
+
+def find_prefixed_labels(labels, prefix):
+  """Util for filtering and cleaning labels that are changelog-relevant.
+
+  Given a list of labels, find only the changelog-specific labels and
+  removes the prefix indicating they are for downstreams.
```
Comment (chrisst) [inline .ci/magic-modules/pyutils/strutils.py]: I don't think it removes the prefix

File: .ci/magic-modules/downstream_changelog_metadata.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  upstream_pr = gh.get_repo(downstreams.UPSTREAM_REPO).get_pull(pr_num)
+  release_note = changelog.get_release_note(upstream_pr.body)
+  labels_to_add = changelog.find_prefixed_labels(
+    [l.name for l in upstream_pr.labels],
+    CHANGELOG_LABEL_PREFIX)
+
+  print "Applying changelog info to downstreams for upstream PR %d:" % (
+    upstream_pr.number)
+  print "Release Note: \"%s\"" % release_note
+  print "Labels: [%s]" % changelog_labels
```
Comment (chrisst) [inline .ci/magic-modules/downstream_changelog_metadata.py]: `labels_to_add` rename didn't get pushed all the way through

File: .ci/magic-modules/downstream_changelog_metadata.py [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  if len(downstream_urls) == 0:
+    puts "Skipping, no downstreams repos given to downstream changelog info for"
+    sys.exit(0)
+  gh = Github(os.environ.get('GITHUB_TOKEN'))
+
+  assert len(sys.argv) == 2, "expected id filename as argument"
+  with open(sys.argv[1]) as f:
+    pr_num = int(f.read())
+
+    # TODO(emilymye): Replace this no-op print statement with code after
```
Comment (chrisst) [inline .ci/magic-modules/downstream_changelog_metadata.py]: :)

File: .ci/magic-modules/pyutils/strutils.py (hunk 2 above)
Comment (emilymye) [reply .ci/magic-modules/pyutils/strutils.py]: i'm adding it back so it does - sorry, wasn't actually asking for review yet :) just wanted you to see my naming change

**Pull request merged**